web/src/components/inventory/ShowNearby.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+import useNuiEvent from '../../hooks/useNuiEvent';
+import { giveTo } from '../../dnd/onGive';
+import { useExitListener } from '../../hooks/useExitListener';
+
+interface Player {
+  player: number;
+}
```

Comment:
Why make an interface with just one number value in it?

Reply:
Ah sorry bout that. Again this is the very first time i've tried typescript, do improve it if i was wrong.

Reply:
Ok fixed it, it should only contain player ids now, not object with player key and player id value.